demos/demo03_poisson_circular.py

```diff
@@ -131,17 +132,18 @@ def heavy_map(func):
 
 
 #%% 3.3 Create pCN Sampler 
-pCNSampler = cuqi.sampler.pCN(
+pCNSampler = cuqi.sampler.PCN(
     posterior,
-    scale=None,
-    x0=None,
+    initial_point=np.zeros(model.domain_dim),
+    scale=0.04
 )
 
 #%% 3.4 Sample using the pCN sampler
-samplespCN = pCNSampler.sample_adapt(10000)
+pCNSampler.sample(50000)
```

Comment:
Why the change in parameter value? That seems unrelated to the issue/PR scope of updating to use new sampler?